vendor/bat-native-ads/src/bat/ads/internal/logging.h

```diff
@@ -1,26 +1,92 @@
-/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #ifndef BAT_ADS_INTERNAL_LOGGING_H_
 #define BAT_ADS_INTERNAL_LOGGING_H_
 
-#include "bat/ads/ads_client.h"
+#include <ostream>
+#include <string>
 
-#define ADS_LOG_INFO \
-  ads_client_->Log(__FILE__, __LINE__, ::ads::LogLevel::LOG_INFO)
+#include "bat/ads/logging.h"
 
-#define ADS_LOG_WARNING \
-  ads_client_->Log(__FILE__, __LINE__, ::ads::LogLevel::LOG_WARNING)
+#include "base/logging.h"
```

Comment:
Required at this time for `CHECK`, `DCHECK` and `NOTREACHED` which is out of scope for this ticket